src/routes/users/[userId].js

```diff
@@ -7,11 +7,13 @@ import {
   getUserById,
   updateInterestChannels,
 } from "../../controllers/userController.js";
+import { updateUserSettings } from "../../controllers/userSettingController.js";
 import { validateUserId } from "../../validators/userValidator.js";
 
 const router = express.Router();
 
 router.get("/:userId", validateUserId, getUserById);
+
```

Comment:
사용자 관련 라우터가 떨어져 있어 응집도가 떨어지는 것 같은데 사용자 관련 라우트는 한 곳에 모아두는건 어떨까요? 아래 관심 채널 관련 라우터가 한 곳에 모여있는 것처럼요! 만약 이렇게 하신다면, 사용자 관련 라우터와 즐겨찾기 관련 라우터 사이에 개행도 있어야 하겠네요!

```suggestion
router.put("/:userId/settings", validateUserId, updateUserSettings);
```

Reply:
사용자 관련 라우터가 나눠져 있어서 흐름이 살짝 끊겼던 것 같네요.
말씀해주신 대로 관심 채널처럼 모아두면 더 깔끔할 것 같아요~
좋은 의견 감사합니다!